### ISSUE TITLE
Month picker navigation using keyboard keys not working

### ISSUE THREAD

**Issue by gawadesantosh14** [Hacker0x01/react-datepicker#2078]
**Describe the bug**
User can not able to select month using keyboard navigation keys. Only the year can allow to change

**To Reproduce**
() => {
  const [startDate, setStartDate] = useState(new Date());
  return (
    <DatePicker
      selected={startDate}
      onChange={date => setStartDate(date)}
      dateFormat="MM/yyyy"
      showMonthYearPicker
    />
  );
};

1. Go to month input
2. Click on keyboard arrow keys

**Expected behavior**
A month should be able to select using keyboard navigation keys

**Comment by natakina** [comment]
Bumped into the same issue.

**Comment by renatoch** [comment]
+1
Same problem here

**Comment by camilaazuma** [comment]
+1
